ui/src/shared/utils/TimeMachineContainer.ts

```diff
@@ -126,6 +128,10 @@ export class TimeMachineContainer extends Container<TimeMachineState> {
     return this.setAndPersistState({script})
   }
 
+  public handleUpdateDraftScript = (draftScript: string) => {
+    return this.setAndPersistState({draftScript})
```

Comment:
Does the draft need to persist?

Reply:
fixed 👍

Reply:
actually I changed it back bc @chnn pointed out that this was to keep the user's changes if they refresh the page